### DISCUSSION TITLE
Cüzdanını kaybedenler için çözüm var mı

### DISCUSSION THREAD

**Discussion by Eddyhunter07** [IBAX-io/go-ibax#99] [Bounty Hunter]
Cüzdanını kaybeden bir sürü insan var Twitter aracılığıyla ulaşılabilir mi acaba ?

**Comment by alisabanoglu** [comment]
Var sanırsam cüzdan olusturup bilgileri saklıyorum

**Comment by hemresenol** [comment]
Müşteri hizmetleriyle iletişime geçin

**Comment by Grekny** [comment]
bence daha erkenken direk yeni wallet aç yeniden yap
